src/payment/PaymentProcessingModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  ModalDialog, Spinner,
+} from '@edx/paragon';
+import { useSelector } from 'react-redux';
+import { useIntl, defineMessages } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+
+// import { subscriptionStatusSelector } from '../data/status/selectors';
```

Comment:
Meant to be paymentStatusSelector as a TODO? or just remove.

Reply:
Ill comment on it and leave a TODO.